Resources/Computing/BatchSystems/TimeLeft/SGEResourceUsage.py

```diff
@@ -57,37 +55,41 @@ def getResourceUsage(self):
         try:
           newcpu = 0.
           if len(cpuList) == 3:
-            newcpu = (float(cpuList[0]) * 60 + float(cpuList[1])) * 60 + float(cpuList[2])
+            newcpu = (float(cpuList[0]) * 3600 + float(cpuList[1])) * 60 + float(cpuList[2])
```

Comment:
Are you sure about replacing `60` with `3600`?
I guess that `cpuList[0]` represents hours and `cpuList[1]` is minutes, right?
In this case if you have 3:05:10:
`newcpu = (3 * 60 + 5) * 60 + 10` seems to provide a more accurate result. Am I wrong?

Reply:
Thanks! Fixes.